script.module.dropbox/lib/dropbox/pkg_resources.py

```diff
@@ -0,0 +1,3 @@
+
+def resource_filename(*args):
```

Comment:
If you want to provide `pkg_resources.resource_filename(...)`, this one has wrong arguments.
Correct is `def resource_filename(package_or_requirement, resource_name):`